crates/solidity/outputs/cargo/tests/src/backend/passes/p1_flatten_contracts.rs

```diff
@@ -0,0 +1,51 @@
+use anyhow::{anyhow, Result};
+use slang_solidity::backend::l2_flat_contracts;
+use slang_solidity::backend::passes::{p0_build_ast, p1_flatten_contracts};
+use slang_solidity::compilation::InternalCompilationBuilder;
+use slang_solidity::utils::LanguageFacts;
+
+#[test]
+fn test_flatten_contracts() -> Result<()> {
```

Comment:
Unless I'm missing something, this test is not very interesting for flattening

Reply:
It's testing what the pass is currently doing, ie. flattening the `ContractSpecifiers` collection into separate collections of `StorageLayoutSpecifier` and `InheritanceSpecifier` elements.